extensions/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/NotFoundExceptionMapper.java

```diff
@@ -266,4 +275,9 @@ public static void staticResources(Set<String> knownFiles) {
     public static void nonJaxRsClassNameToMethodPaths(Map<String, NonJaxRsClassMappings> nonJaxRsPaths) {
         NotFoundExceptionMapper.nonJaxRsClassNameToMethodPaths = nonJaxRsPaths;
     }
+
+    public static void setAdditionalEndpoints(List<String> additionalEndpoints) {
+        Collections.sort(additionalEndpoints);
```

Comment:
The ordering could be moved to build time to avoid bytecode generation and its execution at runtime.

Reply:
Sure, let's do this.